server/migrations/20160106132118-create-robot.js

```diff
@@ -12,7 +12,8 @@ module.exports = {
         type: Sequelize.TEXT
       },
       url_name: {
-        type: Sequelize.TEXT
+        type: Sequelize.TEXT,
+        unique: true
```

Comment:
This should be removed from the old migration and included in the new migration (which it already is)